lib/Dialect/FIRRTL/Export/FIREmitter.cpp

```diff
@@ -1040,6 +1041,8 @@ void Emitter::emitType(Type type, bool includeConst) {
         ps << "Analog";
         emitWidth(type.getWidth());
       })
+      .Case<BaseTypeAliasType>(
+          [&](auto type) { emitType(type.getInnerType()); })
```

Comment:
FIREmitter strips the type alias? Should/could this emit the alias instead, once it's in the FIRRTL spec anyway?